src/styles/_catalog.scss

```
@@ -14,6 +13,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms , color 300ms;
```

Use var for the time value, it's repeated 3 times
```suggestion
    transition: transform $transition-time , color $transition-time;
```